src/coreclr/System.Private.CoreLib/src/System/Runtime/InteropServices/TrackerObjectManager.CoreCLR.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Text;
+
+namespace System.Runtime.InteropServices
+{
+    internal static partial class TrackerObjectManager
+    {
+        private static partial bool HasReferenceTrackerManager
+            => HasReferenceTrackerManagerInternal();
+
+        [LibraryImport(RuntimeHelpers.QCall, EntryPoint = "TrackerObjectManager_HasReferenceTrackerManager")]
+        [SuppressGCTransition]
+        [return: MarshalAs(UnmanagedType.Bool)]
+        private static partial bool HasReferenceTrackerManagerInternal();
```

Comment:
I would change this to return `CLR_BOOL` in the unmanaged side and avoid the marshalling stub entirely.

Reply:
Same through this file when the QCall is marked with `SuppressGCTransition`.

Reply:
As this is a `LibraryImport`, we need to specify some sort of marshalling here (we don't allow plain `bool` types). I can change it to use `U1` to possibly get better code-gen.